packages/orbit-components/playwright-ct.config.ts

```diff
@@ -0,0 +1,64 @@
+import { defineConfig, devices } from "@playwright/experimental-ct-react";
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  testDir: "src",
+  testMatch: "**/*.ct.tsx",
+  outputDir: "playwright-report",
+  timeout: 10 * 1000,
+  fullyParallel: true,
+  forbidOnly: process.env.CI != null,
+  retries: process.env.CI == null ? 0 : 2,
+  workers: process.env.CI == null ? undefined : 1,
+  use: {
+    trace: "on-first-retry",
+    testIdAttribute: "data-test",
+    ctPort: 3100,
+  },
+  projects: [
+    {
+      name: "Small Mobile",
+      use: {
+        ...devices["iPhone SE"],
+        viewport: { ...devices["iPhone SE"].viewport, width: 320 },
```

Comment:
I think you can use token values here for width, so it's not hardcoded :)

Reply:
adjusted